miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -5,6 +5,17 @@ use.miden::kernels::tx::memory
 use.miden::kernels::tx::note
 use.miden::kernels::tx::tx
 
+# ERRORS
+# =================================================================================================
+
+# Starting at 2**31
+
+# For faucets the slot FAUCET_STORAGE_DATA_SLOT is reserved and can not be used with set_account_item
+const.ERR_FAUCET_RESERVED_DATA_SLOT=2147483648
```

Comment:
Since we use error codes based on hexadecimal values, maybe it will be more readable to use them in a hex form? For example:
```text
const.ERR_FAUCET_RESERVED_DATA_SLOT=0x80000000
```

Reply:
changed